bind/uniffi/src/credentials/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod presentation_definition;
+pub mod verifiable_credential_11;
```

Comment:
alternatively 
pub mod verifiable_credential_v1_1;

Reply:
nice I like this too, done